### ISSUE TITLE
bash: cat in variable assignment -> redirect?

### ISSUE THREAD

**Issue by sburris0** [koalaman/shellcheck#2493]
#### For new checks and feature suggestions
- [x] https://www.shellcheck.net/ (i.e. the latest commit) currently gives no useful warnings about this
- [x] I searched through https://github.com/koalaman/shellcheck/issues and didn't find anything related

While reading through the bash manual I encountered this:
```
The command substitution $(cat file) can be replaced by the equivalent but faster $(< file). 
```

It's a relatively minor thing but it might be worth suggesting to users.
Thoughts?

#### Here's a snippet or screenshot that shows the problem:

```sh
#!/bin/bash
var=$(cat file.txt)
```

#### Here's what shellcheck currently says:
n/a

#### Here's what I wanted or expected to see:
Suggestion to use `var=$(< file.txt)` instead
